hvm/config/config.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import (
+    "encoding/json"
+    "os"
+    "github.com/pkg/errors"
+    "path"
+)
+
+type HvmConfig struct {
+
+    GithubPAT *string
+
+}
+
+
+func GetHvmConfig() (*HvmConfig, error){
+	home, err := os.UserHomeDir()
```

Comment:
can you install a golang formatter for your editor, it looks like you have some formatting to do on these line. Or just run `go fmt` on the project